### PR TITLE
Fix: typos and errors

### PR DESCRIPTION
**Pull request by Raj-Pansuriya**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes no issue

### CODE REVIEW

**Comment by Raj-Pansuriya** [comment]
In the last  bit of this tutorial, we were asked to view log without any flags but the test fails when we do that.
The test passes when we do use `--oneline` flag instead. Can you point me in the correct direction on how to fix this test?
image

File: TUTORIAL.md [hunk 1]
```diff
@@ -2743,7 +2743,7 @@ Both, the commit to add the unique command and the one to revert it were dropped
 
 ### 1570.1
 
-You can see that the latest commit is at the bottom here. Be careful not to change the wrong commits. One of the options is `r, reword = use commit, but edit the commit message`. Replace `pick` with an `r` next to the commit with the message `feat: add column reference` to reword the message, it's the very first commit you added to this branch. When you are done, save the file and exit Nano.
+You can see that the latest commit is at the bottom here. Be careful not to change the wrong commits. One of the options is `r, reword = use commit, but edit the commit message`. Replace `pick` with an `r` next to the commit with the message `feat: add column reference` to reword the message, it's the very first commit you added to this branch. When you are done, save the file and exit Nano. Git will put you in an another `nano` instance to `reword` commit message for the commit you just chose. Do not change anything in it just yet.
```
Comment (moT01) [inline TUTORIAL.md]:
```suggestion
You can see that the latest commit is at the bottom here. Be careful not to change the wrong commits. One of the options is `r, reword = use commit, but edit the commit message`. Replace `pick` with an `r` next to the commit with the message `feat: add column reference` to reword the message, it's the very first commit you added to this branch. When you are done, save the file and exit Nano. Git will put you in another Nano instance to reword the commit message. Don't change anything in it yet.
```

File: TUTORIAL.md [hunk 2]
```diff
@@ -2819,7 +2819,7 @@ Now the hashes are the same as they were before you rebased back to `--root`, wh
 
 ### 1590.1
 
-Squashing commits means that you will take a bunch of commits and turn them into one. This is helpful to keep your commit history clean and something you want try to do. Replace `pick` with an `s` next to all your commits except the one with the message `feat: add column references`. When you are done, save and exit the file.
+Squashing commits means that you will take a bunch of commits and turn them into one. This is helpful to keep your commit history clean and something you want try to do. Replace `pick` with an `s` next to all your commits except the one with the message `feat: add column references`. When you are done, save and exit the file. You will find yourself in another instance of `nano`. Don't change aything just yet.
```
Comment (moT01) [inline TUTORIAL.md]:
```suggestion
Squashing commits means that you will take a bunch of commits and turn them into one. This is helpful to keep your commit history clean and something you want try to do. Replace `pick` with an `s` next to all your commits except the one with the message `feat: add column references`. When you are done, save and exit the file. You will find yourself in another instance of Nano. Don't change anything in it yet.
```

File: TUTORIAL.md [hunk 3]
```diff
@@ -2755,7 +2755,7 @@ You can see that the latest commit is at the bottom here. Be careful not to chan
 
 ### 1572.1
 
-Add an `s` at the end of the commit message so it is `feat: add column references`. When you are done, save the file and exit Nano.
+You are already in an `nano` instance, editing the commit meassage for the selected commit. Add an `s` at the end of the commit message so it is `feat: add column references`. When you are done, save the file and exit Nano.
```
Comment (moT01) [inline TUTORIAL.md]:
```suggestion
Git is waiting for you to submit the commit message. Add an `s` at the end of the commit message so it is `feat: add column references`. When you are done, save the file and exit Nano.
```

**Review by moT01 — COMMENTED** [review]
This is looking good @Raj-Pansuriya - We want to keep the instructions as short as possible. I made some suggestions shorten them a little bit and fix some typos.

**Comment by Raj-Pansuriya** [comment]
Done with all the requested changes.

**Comment by Raj-Pansuriya** [comment]
> In the last bit of this tutorial, we were asked to view log without any flags but the test fails when we do that. The test passes when we do use `--oneline` flag instead. Can you point me in the correct direction on how to fix this test? image

Can you please guide me to fix this test. You see there is no issues tab so I have to communicate using PRs

**Comment by gikf** [comment]
@Raj-Pansuriya issues for these tutorials can be opened in `freeCodeCamp/freeCodeCamp` repository - https://github.com/freeCodeCamp/freeCodeCamp/issues. General guidelines for changing tests are available here: https://contribute.freecodecamp.org/#/how-to-work-on-tutorials-that-use-coderoad?id=working-on-version-branch.

**Comment by Raj-Pansuriya** [comment]
ok thanks for the guidelines

**Comment by Raj-Pansuriya** [comment]
> Done with all the requested changes.

I have done all the requested changes, you can just check if the PR is valid for merge

**Review by moT01 — APPROVED** [review]
Sorry for the delay on this @Raj-Pansuriya. I merged a different PR that made one conflict here so I had to give it a quick rebase. Thanks for working on this and happy coding 🎉

**Comment by Raj-Pansuriya** [comment]
> Sorry for the delay on this @Raj-Pansuriya. I merged a different PR that made one conflict here so I had to give it a quick rebase. Thanks for working on this and happy coding tada

No problem at  all Tom. Happy to contribute!!

**Pull request merged**